dotnet/TF2SA.Common/Models/LogsTF/GameLogModel/LogInfo.cs

```diff
@@ -9,58 +9,59 @@ public class LogInfo : IJsonOnDeserialized
 	public string? Map { get; set; }
 
 	[JsonPropertyName("supplemental")]
-	public bool? IsSupplemental { get; set; }
+	public bool? IsSupplemental { get; set; } = false;
```

Comment:
just initialize to false.
sometimes these are null from logstf, so assume that the server doesn't track